adminSelection.php

```diff
@@ -0,0 +1,45 @@
+<?php include("components/header.php");?>
+
```

Comment:
Vue not integrated > make a div with an id of "v_adminSelection" then create a vue js file inside vue folder. 
link that vue js file on our FE Scripts. then try reloading the page.